### ISSUE TITLE
Bug: Card Template Not Carrying All Checkboxes Into New Card on Boards

### ISSUE THREAD

**Issue by jtb-jtb** [mattermost/focalboard#3704]
## Steps to reproduce the behavior

1. Create Card Template
2. Add multiple checkboxes at bottom of card
3. Exit out of card template editor
4. Create new card with template
5. See only first checkbox is present on new card
## Expected behavior

The new card should have all checkboxes from the template included when a new card is created

## Screenshots (optional)

![image](https://user-images.githubusercontent.com/111531635/185468720-b73854b4-49b1-4db0-866f-e5345ee7ebe9.png)
![image](https://user-images.githubusercontent.com/111531635/185468834-ccd742a2-bae5-4c03-96ff-f44481e6ca81.png)

## Edition and Platform

 - Edition: Personal Desktop / Cloud Server / Mattermost Boards (plugin)
 - Version: v7.2
 - Browser and OS: Chrome on Windows]

## Additional context (optional)

* Sev 2: Affects major functionality with a difficult or non-obvious workaround

**Comment by wuwinson** [comment]
@jtb-jtb We can't repro this on [Community](https://community.mattermost.com/boards/) with the latest v7.2 release. Can you try to upgrade and see if this is still happening for you? Thanks!

**Comment by jtb-jtb** [comment]
We're on cloud and I verified the plugin is v7.2. I tried it on Community as well, and it gave me the same results :( . I created a card template, added a checkbox at the bottom (not on top with the properties but below the comments), then added a couple more. Exited the template manager, created a card from that template, and only the first checkbox is there. If I open the template the checkboxes are there, but it doesn't create them when creating a new card.
